docs/fedora-releases-guide/index.md

```diff
@@ -217,26 +217,57 @@ for overriding the Packit default behaviour, for example:
   - for **source archive(s) generation**, you can utilise e.g. `pre-sync` action, see 
   [this example](/docs/configuration/examples#custom-archive-creation-for-release-syncing)
 
-#### Keeping dist-git branches non-divergent
-Packit currently syncs the release in a way that the branches become divergent (you can follow 
-the request to change this behaviour [here](https://github.com/packit/packit/issues/1724)). 
-
-However, if you wish to keep your dist-git branches in sync, you can configure Packit to propose updates exclusively 
-to `rawhide` (by specifying `dist_git_branches: fedora-rawhide`) and you can locally merge it with the stable release branches. 
-The following example demonstrates how to achieve this for a single branch (`f39` in this case):
-```bash
-# Clone the dist-git repository if you haven't done so already
-fedpkg clone $PACKAGE
-# or
-git clone ssh://$YOUR_USER@pkgs.fedoraproject.org/rpms/$PACKAGE.git
-
-# Alternatively, pull the rawhide changes only
-git pull origin rawhide
-
-# Switch to the desired branch and merge it with the updated rawhide branch
-git checkout f39
-git merge rawhide
-git push origin f39
+#### Keep dist-git branches non-divergent
+
+If you want to keep your dist-git branches from diverging,
+you can use the new `dist_git_branches` syntax:
+  
+```yaml
+  dist_git_branches:
+    rawhide:
+      fast_forward_merge_into: [fedora-branched]
+    epel-9: {}    
+      
+```
+
+In this example, Packit runs the downstream synchronization just for the
+`rawhide` and `epel-9` branches and opens a new pull request for every
+`fedora-branched` branch trying to fast forward the changes introduced
+in the `rawhide` branch.
```

Comment:
I am not sure if this is still easy to understand but I see this is really hard to describe simply...;)

```suggestion
In this example, Packit runs the downstream synchronization process for the
`rawhide` and `epel-9` branches as usual. But Packit also opens a new pull request reusing the commit from `rawhide` for every
`fedora-branched` branch so it can be fast-forwarded when merging.
```